SwanContents/swancontents/checkpoints/eoscheckpoints.py

```diff
@@ -91,12 +92,12 @@ def delete_checkpoint(self, checkpoint_id, path):
         with self.perm_to_403():
```

Comment:
What's the rational behind making the creation of the checkpoint async but leaving the deletion sync?

Reply:
And a similar comment for restore_checkpoint.

Reply:
I changed the other methods to be async as well